ledger/controllers/wallet.js

```diff
@@ -295,12 +298,16 @@ const write = function (runtime, apiVersion) {
       return reply(boom.badData(ex.toString()))
     }
 
+    params = surveyor.payload.adFree
+    txnProbi = runtime.wallet.getTxProbi(wallet, txn)
+    totalVotes = txnProbi.dividedBy(params.probi).times(params.votes).round().toNumber()
+
```

Comment:
can we add a sanity check here to ensure `totalVotes >= 1` and error if not?

Reply:
https://github.com/brave-intl/bat-ledger/pull/516/commits/5324e8c0dd75d8fb6145ffadc68ba0874e077d63